### PR TITLE
Update package versions to 1.1.0

### PR DESCRIPTION
**Pull request by gkhanna79**
@weshaggard @russellhadley @RussKeldorph PTAL

CC @leecow @dagood

### CODE REVIEW

**Comment by RussKeldorph** [comment]
Changing every .pkgproj file in the tree to Version=1.1.0.  LGTM.

**Comment by gkhanna79** [comment]
@dotnet-bot test Ubuntu x64 Checked Build and Test please

**Comment by gkhanna79** [comment]
@dotnet-bot test Linux ARM Emulator Cross Release Build please

**Comment by gkhanna79** [comment]
@weshaggard @leecow Are you good with this change?

**Comment by leecow** [comment]
LGTM

File: src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.pkgproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
-    <Version>1.0.4</Version>
+    <Version>1.1.0</Version>
     <SkipPackageFileCheck>true</SkipPackageFileCheck>
     <SkipValidatePackage>true</SkipValidatePackage>
```
Comment (weshaggard) [inline src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.pkgproj]: You will need to update the dependency package versions in this project once those are updated.

Comment (gkhanna79) [inline src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.pkgproj]: I am not updating any other dependency version (JIT is referenced via ProjectReference and I am not changing APISet dependency version). Is there a different dependency you are referring to?

Comment (weshaggard) [inline src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.pkgproj]: I was referring to the arm package and the JIT package, which will need to be updated once we have updated versions of them.

Comment (gkhanna79) [inline src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.pkgproj]: I have fixed the Arm package reference. JIT is referred to by ProjectReference (https://github.com/dotnet/coreclr/pull/6657/files#diff-341f08888400e32a087d8ca1bbe0d04cL23). Thus, that is auto-updated as dependency. Is that not correct expectation?

Comment (weshaggard) [inline src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.pkgproj]: Ah yes the JIT reference is fine, it was only in the release branch where it was a specific version.

**Comment by gkhanna79** [comment]
@dotnet-bot test FreeBSD x64 Checked Build please

**Comment by gkhanna79** [comment]
@dotnet-bot test Ubuntu x64 Checked Build and Test please

**Comment by gkhanna79** [comment]
@dotnet-bot test Windows_NT x86 ryujit Checked Build and Test please

**Pull request merged**